mlflow/server/handlers.py

```diff
@@ -27,9 +29,13 @@ def _get_store():
     from mlflow.server import FILE_STORE_ENV_VAR, ARTIFACT_ROOT_ENV_VAR
     global _store
     if _store is None:
-        store_dir = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
-        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, store_dir)
-        _store = FileStore(store_dir, artifact_root)
+        store_uri = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
```

Comment:
In utils there is special function for that. I think it's preferable https://github.com/mlflow/mlflow/blob/master/mlflow/utils/env.py#L4